composer-json-php-ext-schema.json

```
@@ -34,6 +34,15 @@
                     "example": false,
                     "default": true
                 },
+                "os-families-exclude": {
+                    "type": "array",
+                    "description": "An array of OS families to mark as incompatible with the extension.",
+                    "items": {
+                        "type": "string",
```

I would suggest we restrict the values according to the [PHP docs](https://www.php.net/manual/en/reserved.constants.php#constant.php-os-family), e.g.:

```suggestion
                        "type": "string",
                        "enum": ["Windows", "BSD", "Darwin", "Solaris", "Linux", "Unknown"],
```

---

Indeed, that's way better!